.kokoro

```diff
@@ -49,6 +49,7 @@ fix_bazel_imports() {
   rewrite_tests "s/import MaterialComponents.Material(.+)/import components_\1_\1/"
   rewrite_source "s/import <Motion(.+)\/Motion.+\.h>/import \"Motion\1.h\"/"
   rewrite_source "s/import <MDF(.+)\/MDF.+\.h>/import \"MDF\1.h\"/"
+  rewrite_source "s/import <MaterialComponents\/Material(.+)\.h>/import\/\*framework import\*\/ \"Material\1.h\"/"
```

Comment:
Huh, this and [this](https://github.com/material-components/material-components-ios/pull/5717/files#diff-9b8844c11cf1465b9e2ab4e4a08e757cR67) were added in https://github.com/material-components/material-components-ios/pull/5541. Should they be here?

Reply:
This was a WIP branch. I imported, modified, then exported develop. Apparently that flow isn't idempotent.